site2/website/release-notes.md

```diff
@@ -1,6 +1,300 @@
+- Align configurations defaults between default file and Java object (broker.conf, proxy.conf, websocket.conf) [#13272](https://github.com/apache/pulsar/pull/13272)
+- [PIP 118] Do not restart brokers when ZooKeeper session expires as default [#13341](https://github.com/apache/pulsar/pull/13341)
+- [PIP 119] Enabled consistent hashing by default on KeyShared subscriptions dispatcher [#13352](https://github.com/apache/pulsar/pull/13352)
+- [PIP 120] Enabled client memory limit controller by default [#13344](https://github.com/apache/pulsar/pull/13344)
+- Make Docker images non-root, by default, and OpenShift compliant [#13376](https://github.com/apache/pulsar/pull/13376)
+- [PIP 122] Change loadBalancer default loadSheddingStrategy to ThresholdShedder [#13733](https://github.com/apache/pulsar/pull/13733)
+- Fix netcat returning early for probe [#14088](https://github.com/apache/pulsar/pull/14088)
+
+### PIPs
+- [PIP 79] Added lazy-loading feature to PartitionedProducer [#10279](https://github.com/apache/pulsar/pull/10279)
+- [PIP 84] Pulsar client: Redeliver command add epoch [#10478](https://github.com/apache/pulsar/pull/10478)
+- [PIP 86] Pulsar Functions: Preload and release external resources [#13205](https://github.com/apache/pulsar/pull/13205)
+- [PIP 97] Update Authentication Interfaces to Include Async Authentication Methods [#12104](https://github.com/apache/pulsar/pull/12104)
+- [PIP 92] Topic policy across multiple clusters [#12517](https://github.com/apache/pulsar/pull/12517)
+- [PIP 104] Add new consumer type: TableView [#12838](https://github.com/apache/pulsar/pull/12838)
+- [PIP-105] Support pluggable entry filter in Dispatcher [#12869](https://github.com/apache/pulsar/pull/12869) [#12970](https://github.com/apache/pulsar/pull/12970) [#12979](https://github.com/apache/pulsar/pull/12979)
+- [PIP 106] Broker extensions to provide operators of enterprise-wide clusters better control and flexibility [#12536](https://github.com/apache/pulsar/pull/12536)
+- [PIP 107] Introduce chunk message ID [#12403](https://github.com/apache/pulsar/pull/12403)
+- [PIP 110] Support Topic metadata - PART-1  create topic with properties [#12818](https://github.com/apache/pulsar/pull/12818)
+- [PIP 121] Pulsar cluster level auto failover on client side [#13316](https://github.com/apache/pulsar/pull/13316)
+- [PIP-124] Create init subscription before sending message to DLQ  [#13355](https://github.com/apache/pulsar/pull/13355)
+- [PIP-130] Apply redelivery backoff policy for ack timeout [#13707](https://github.com/apache/pulsar/pull/13707)
+- [PIP 131] Resolve produce chunk messages failed when topic level maxMessageSize is set [#13599](https://github.com/apache/pulsar/pull/13599)
+- [PIP 135] Include MetadataStore backend for Etcd [#13225](https://github.com/apache/pulsar/pull/13225)
+
+### Broker
+- [PIP 45] Added BookKeeper metadata adapter based on MetadataStore [#12770](https://github.com/apache/pulsar/pull/12770)
```

Comment:
I think it makes sense to group all of the "[PIP 45]" fixes into sub bullet points.

- [PIP 45]
  - ...
  - ...